src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/pensjon/HistoriskPensjonController.kt

```diff
@@ -0,0 +1,21 @@
+package no.nav.familie.ef.sak.opplysninger.personopplysninger.pensjon
+
+import no.nav.familie.kontrakter.felles.Ressurs
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.cache.annotation.Cacheable
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping(path = ["/api/historiskpensjon"])
+@ProtectedWithClaims(issuer = "azuread")
+class HistoriskPensjonController(val historiskPensjonService: HistoriskPensjonService) {
+
+    @GetMapping("{personIdent}")
+    @Cacheable("historisk_pensjon")
```

Comment:
Burde vi flytte cachen til servicen? 
Den trenger egentlige ikke noe navn, her settes navn på cachen, og bruker default cache
For å bruke spesifik cacheManager så må man bruke
`@CacheConfig(cacheManager = "<cachemanager>")`

Reply:
Ser nå at servicen også er cachet, er det noen grunn til å sette cache her og?

Reply:
Fjernet cacheable annotering i controlleren